.github/workflows/index.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
           echo "RELEASE_COMMIT=$(git -C llvm-project rev-parse HEAD)" >> $GITHUB_ENV
           echo "RELEASE_COMMIT_SHORT=$(printf '%.12s' $(git -C llvm-project rev-parse HEAD))" >> $GITHUB_ENV
           echo "RELEASE_DATE=$(date -u +%Y%m%d)" >> $GITHUB_ENV
+          echo "RELEASE_NAME=index_$(date -u +%Y%m%d)" >> $GITHUB_ENV
```

Comment:
For tags, i think `/` is a more conventional separator than  `_` (don't remember why we used underscore for clangd release)